### PR TITLE
Add number of available tickets per fare product

### PR DESCRIPTION
**Pull request by ue71603**

### CODE REVIEW

**Comment by normanoffel** [comment]
what is the use case? as in a webshop "5 tickets remaining"? where exactly is the change? I assume it's AvailableTicketsAtPrice in FareSupport.xsd in the FareProductStructure. It's optional and therefore ok for me.

**Review by normanoffel — APPROVED** [review]

File: OJP/OJP_FareSupport.xsd [hunk 1]
```diff
@@ -556,6 +556,11 @@
 				</xs:annotation>
 			</xs:element>
 			<xs:group ref="FareProductPriceGroup"/>
+			<xs:element name="AvailableTicketsAtPrice" type="xs:positiveInteger" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Available tickets at the given price. When absent, the number of remaining tickets is unknown or it is not limited.</xs:documentation>
```
Comment (ue71603) [inline OJP/OJP_FareSupport.xsd:561]:
```suggestion
					<xs:documentation>Remaining offered tickets in this FARE PRODUCT. When absent, the number of remaining tickets is unknown or it is not limited.</xs:documentation>
```

File: OJP/OJP_FareSupport.xsd [hunk 2]
```diff
@@ -556,6 +556,11 @@
 				</xs:annotation>
 			</xs:element>
 			<xs:group ref="FareProductPriceGroup"/>
+			<xs:element name="AvailableTicketsAtPrice" type="xs:positiveInteger" minOccurs="0">
```
Comment (skinkie) [inline OJP/OJP_FareSupport.xsd:559]: This means that 0 is not available. With zero you don't show this ticket, as in you won't tell (ever) that it is unavailable?

Comment (Aurige) [inline OJP/OJP_FareSupport.xsd:559]: The Name is FareQuota in Transmodel/NeTEx
image

Comment (skinkie) [inline OJP/OJP_FareSupport.xsd:559]:
```suggestion
			<xs:element name="FairQuota" type="xs:integer" minOccurs="0">
```

File: OJP/OJP_FareSupport.xsd [hunk 3]
```diff
@@ -556,6 +556,11 @@
 				</xs:annotation>
 			</xs:element>
 			<xs:group ref="FareProductPriceGroup"/>
+			<xs:element name="FareQuota" type="xs:integer" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Remaining offered tickets in this FARE PRODUCT. When absent, the number of remaining tickets is unknown or it is not limited.</xs:documentation>
```
Comment (ue71603) [inline OJP/OJP_FareSupport.xsd:561]:
```suggestion
					<xs:documentation>Remaining offered tickets in this FARE PRODUCT. When absent, the number of remaining tickets is unknown or it is not limited. In most cases if the FareQuota is zero then the FareProduct is not shown.</xs:documentation>
```

File: examples/functions/fare/FareResponse.xml [hunk 4]
```diff
@@ -38,6 +38,7 @@
 							<FareAuthorityText>Mybuxi</FareAuthorityText>
 							<NetPrice>15</NetPrice>
 							<Currency>CHF</Currency>
+							<AvailableTicketsAtPrice>20</AvailableTicketsAtPrice>
```
Comment (ue71603) [inline examples/functions/fare/FareResponse.xml:41]:
```suggestion
							<FareQuota>20</FareQuota>
```

**Review by Aurige — APPROVED** [review]

**Review by skinkie — APPROVED** [review]

**Pull request merged**